x-pack/plugins/maps/public/shared/layers/sources/wms_source/wms_client.test.js

```diff
@@ -195,4 +195,13 @@ describe('getCapabilities', () => {
       { label: 'layer1', value: '1' },
     ]);
   });
+
+  it('Should not overwrite specific query parameters when defined in the url', async () => {
+    const urlWithQuery = 'http://example.com/wms?format=image/jpeg&service=NotWMS&version=0&request=GetNull&srs=Invalid&transparent=false&width=1024&height=640';
```

Comment:
Can you add a query parameter that is not a default parameter to ensure the query parameters are added to the URL as well?